pandas/core/arrays/arrow/array.py

```diff
@@ -433,11 +433,15 @@ def __abs__(self) -> Self:
     # https://issues.apache.org/jira/browse/ARROW-10739 is addressed
     def __getstate__(self):
         state = self.__dict__.copy()
-        state["_data"] = self._pa_array.combine_chunks()
+        state["_pa_array"] = self._pa_array.combine_chunks()
         return state
 
     def __setstate__(self, state) -> None:
-        state["_pa_array"] = pa.chunked_array(state["_data"])
+        if "_data" in state:
+            data = state["_data"]
```

Comment:
i think pop it here to avoid including it in `__dict__.update` below?

Reply:
Ah good call. Thanks!